### PR TITLE
Toast Component 

### PR DESCRIPTION
**Pull request by bajajcodes**
- Toast Components consists of three options 
1. Simple message toast.
2. Toast with close button.
4. Toast with message, description and close button.

**FILES TO CHECK** 
> css files only

### CODE REVIEW

**Review by raghav2110 — APPROVED** [review]
Good naming for classes. Good looks clean. Good use of semantic HTML tags. Good to merge.

File: css/components/toast.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  width: 100%;
+  max-width: calc(var(--size-9) * 20);
+  background: rgba(255, 255, 255, 0.85);
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  font-size: var(--size-9);
+  user-select: none;
+  row-gap: var(--size-3);
+  border-radius: var(--size-3);
+  background: var(--black);
+  color: yellow;
```
Comment (yaswanthmyneni) [inline css/components/toast.css:13]: changes: use css variables for colors

File: css/components/toast.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.toast {
+  /* position: fixed;
+  bottom: var(--size-10); */
```
Comment (yaswanthmyneni) [inline css/components/toast.css:3]: changes: remove unused code

File: css/components/toast.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+.toast-header {
+  display: flex;
+  align-items: center;
+  padding: var(--size-3) var(--size-7);
+  color: #6c757d;
+  background-color: rgba(255, 255, 255, 0.85);
+  background-clip: padding-box;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+  color: black;
```
Comment (yaswanthmyneni) [inline css/components/toast.css:24]: changes: same here

File: css/components/toast.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  background: var(--black);
+  color: yellow;
```
Comment (yaswanthmyneni) [inline css/components/toast.css:26]: changes: same here

File: css/components/toast.css [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  border-radius: var(--size-3);
+  background: var(--black);
+  color: yellow;
+}
+
+.toast-header {
+  display: flex;
+  align-items: center;
+  padding: var(--size-3) var(--size-7);
+  color: #6c757d;
```
Comment (yaswanthmyneni) [inline css/components/toast.css:20]: changes: same here

File: css/components/toast.css [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.toast-header-button {
+  background: none;
+  border: 0;
+  outline: 0;
+  font-size: var(--size-11);
+  cursor: pointer;
+  margin-left: auto;
+  color: yellow;
```
Comment (yaswanthmyneni) [inline css/components/toast.css:36]: changes: change every where

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes

**Pull request merged**